### PR TITLE
[MS] Update organization filter

### LINKED ISSUE #6122: [🦋 Final Touch] Update organization filter
Missing space between options
Image

### PR DESCRIPTION
**Pull request by fabienSvtr**
### Before
image

### After
image

## Checklist

Before you submit this pull request, please make sure to:

- [x] Keep changes in the pull request as small as possible
   
- [x] Ensure the commit history is sanitized
   
- [x] Give a meaningful title to your PR
- [x] Describe your changes
   
- [x] Link any related issue in the description
   
- [x] Link any dependent pull request in the description

closes #6122

### CODE REVIEW

**Comment by Ironicbay** [comment]
Unsure if that's a relevant point since it's a very specific behaviour, but if you open the popover after reducing the window size enough that it will be cropped, its size can grow as big as the window when the latter is stretched

https://github.com/Scille/parsec-cloud/assets/41511029/d4a9ce70-54ef-4a5b-bde4-31977135ba6d

**Comment by Ironicbay** [comment]
The position also shifts when resizing the window
Screenshot 2024-01-15 at 11 31 46

File: client/src/components/core/ms-dropdown/MsDropdown.vue [hunk 1]
```diff
@@ -109,16 +109,6 @@ function getIcon(): string {
...
-.option {
-  --background-hover: var(--parsec-color-light-primary-50);
-  --color-hover: var(--ion-color-tertiary);
-
-  &.selected {
-    color: var(--parsec-color-light-primary-700);
-    font-weight: bold;
-  }
-}
-
```
Comment (NicoTuxx) [inline client/src/components/core/ms-dropdown/MsDropdown.vue:121]: why remove this?

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/components/core/ms-dropdown/MsDropdown.vue (hunk 1 above)
Comment (fabienSvtr) [reply client/src/components/core/ms-dropdown/MsDropdown.vue:121]: I removed this code because it was unused.

**Comment by fabienSvtr** [comment]
> The position also shifts when resizing the window Screenshot 2024-01-15 at 11 31 46

I open an issue for this: https://github.com/Scille/parsec-cloud/issues/6166

**Review by NicoTuxx — APPROVED** [review]

**Pull request merged**